src/lib/y2partitioner/device_graphs.rb

```diff
@@ -0,0 +1,47 @@
+require "singleton"
+
+module Y2Partitioner
+  # Object that holds and manipulates of storage device graphs.
+  class DeviceGraphs
```

Comment:
@ancorgs `the_partition` is not defined, sorry, I don't understand

Also, I thought we had already agreed that we CANNOT pass partition objects around, because if the partition is going to be formatted, that means adding a FS object to its containing graph, BUT a partition does not KNOW its containing graph. Therefore, we can only pass the graph (I mean, refer to the global/singleton `current` graph) and the partition name to look up by.